buildpipeline/DotNet-CLI-CentOS-x64.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
Yes, that's the long-term plan. I wanted to get this in quickly by doing the work iteratively; I'll log an issue for consolidating the build definitions.